shortix.sh

```diff
@@ -18,7 +15,7 @@ shortix_script () {
     elif [ "$(command -v $PROTONTRICKS_FLAT)" ]; then
         PROTONTRICKS=$PROTONTRICKS_FLAT
     else
-        read -p "Protontricks could not be found! Please install it. Press ENTER to exit."
```

Comment:
I noticed that sometimes the terminal window closes, if the script gets executed manually. This was my attempt to provide a bit of error information if someone is wondering why nothing happened - this won't work if it's run as a service of course

Reply:
I am executing the script from shell/terminal so the window doesn't actually close.
It is not that common to have "blocking" messages in scripts. But I understand. Perhaps the middle ground would be to replace the two error messages and done message with read -p ... -t 10 argument - the read timeout in seconds.